### ISSUE TITLE
A11y/Usability: Product slider - Missing slide controls (unlike Image Slider)

### ISSUE THREAD

**Issue by tieupmedia** [shopware/platform#5501]
*PHP Version:* 8.2   | *Shopware Version:* 6.6.8.2  | *Affected area / extension:* Platform(Default) 

----

+*We decided to improve the current options (arrow options) with this ticket and created a design ticket for a dot navigation / pagination solution.*+

*_Expected behaviour:_*

1. Slide controls (dots/squares/dashes) can be activated in the administration and are displayed in the storefront, where they allow to switch the whole slide (all visible elements) instead of shifting one element forward/backward like the arrow nav does.
2. The admin setting for activating the slide controls looks like in the image slider block.
3. In the storefront slide controls of the product slider look like the image slider by default.

*_Actual behaviour:_*

There is no option, to activate slider controls in the admin settings for the product slider.

![Image]([https://github.com/user-attachments/assets/f9826972-a2b8-4d8c-b5ae-7dbbbf442c9e|https://github.com/user-attachments/assets/f9826972-a2b8-4d8c-b5ae-7dbbbf442c9e])

*_How to reproduce:_*

Go to the admin, choose a shopping experience and setup a product slider vs. an image slider.

!image-20241125-085310.png|width=861,height=536,alt="image-20241125-085310.png"!

**Comment by shopwareBot** [comment]
[public] Shopware automated response: This issue is linked to the internal issue: **NEXT-39635**



[created from NEXT-39635, comment 512972]

**Comment by shopwareBot** [comment]
[public] 

 > FixVersion Change: **6.6.10.0**

 This issue will be / was solved in Shopware version ***6.6.10.0***.



[created from NEXT-39635, comment 520969]
